src/Stripe/Services/Invoices/StripeInvoiceService.cs

```diff
@@ -63,10 +64,11 @@ public virtual IEnumerable<StripeInvoice> List(StripeInvoiceListOptions listOpti
             return Mapper<StripeInvoice>.MapCollectionFromJson(response);
         }
 
-        public virtual StripeInvoice Create(string customerId)
+        public virtual StripeInvoice Create(string customerId, StripeInvoiceCreateOptions createOptions = null)
```

Comment:
I'd suggest creating a separate overload for Create. One that takes the customerId (for backcompat) and one that takes only the createOptions. That way there isn't a duplicate CustomerId and this method follows convention of the other Create methods in services...

...or maybe remove CustomerId from StripeInvoiceCreateOptions.